controllers/ibmpowervscluster_controller_test.go

```diff
@@ -433,6 +441,341 @@ func TestIBMPowerVSClusterReconciler_delete(t *testing.T) {
 	})
 }
 
+func TestReconcileVPCResources(t *testing.T) {
+	testCases := []struct {
+		name                    string
+		powerVSClusterScopeFunc func() *scope.PowerVSClusterScope
+		reconcileResult         reconcileResult
+		conditions              capiv1beta1.Conditions
+	}{
+		{
+			name: "when ReconcileVPC returns error",
+			powerVSClusterScopeFunc: func() *scope.PowerVSClusterScope {
+				clusterScope := &scope.PowerVSClusterScope{
+					IBMPowerVSCluster: &infrav1beta2.IBMPowerVSCluster{},
+				}
+				mockVPC := mock.NewMockVpc(gomock.NewController(t))
+				mockVPC.EXPECT().GetVPCByName(gomock.Any()).Return(nil, errors.New("vpc not found"))
+				clusterScope.IBMVPCClient = mockVPC
+				return clusterScope
+			},
+			reconcileResult: reconcileResult{
+				error: errors.New("vpc not found"),
+			},
+			conditions: capiv1beta1.Conditions{
+				capiv1beta1.Condition{
+					Type:               infrav1beta2.VPCReadyCondition,
+					Status:             "False",
+					Severity:           capiv1beta1.ConditionSeverityError,
+					LastTransitionTime: metav1.Time{},
+					Reason:             infrav1beta2.VPCReconciliationFailedReason,
+					Message:            "vpc not found",
+				},
+			},
+		},
+		{
+			name: "when ReconcileVPC returns requeue",
```

Comment:
```suggestion
			name: "when ReconcileVPC returns requeue as true",
```
Almost every reconcile returns requeue, better to change statement wherever applicable.

Reply:
Updated as suggested